docker-compose.yml

```diff
@@ -7,5 +7,6 @@ services:
     volumes:
```

Comment:
(on line 5): we want to remove the "build", here, because we don't want it in production, and use an image name instead. For image name, we use something like `off-facets-knowledge-panels-api:${VERSION:-dev}`

For prod, we will use an image that we will have compiled with github actions and push to the github docker registry (aka packages).

Reply:
Yes for now, in dev you should always use docker/dev.yml